src/tests/settings/settings.test.js

```diff
@@ -73,3 +76,54 @@ describe('Test for super admin reset user role:', () => {
       });
   });
 });
+
+describe('Test assigning requester to manager', () => {
```

Comment:
@gadishimwe is there a test case that tests the authorization of the user trying to make assignments?

Reply:
No, let me add it.